common/v2/providers/NotificationsProvider/helpers.ts

```diff
@@ -1,3 +1,6 @@
+import { ExtendedNotification } from 'v2/services';
+import moment from 'moment';
```

Comment:
What do you think of moving to `date-fns` instead?
Smaller bundle size + tree-shaking.
 
<img width="675" alt="Screenshot 2019-07-18 at 15 02 17" src="https://user-images.githubusercontent.com/2429708/61459521-2e032c00-a96d-11e9-92ae-27d42f65b417.png">

Reply:
Never heard of it but looks good. Since `moment` is used not only in this PR but also in some reducers that we will get rid of it, I think it would be better to do the transition in a new branch.

Reply:
Ok.